app/views/admin/users/index.haml

```diff
@@ -64,6 +64,23 @@
                   %div= "Sign-in provided by #{user.provider.upcase}"
                 = render 'user_invitation_status', user: user
               %td
+                - if user.invitation_status != :active or user.access_locked?
```

Comment:
this should be `||` not `or`.  Basically don't use `or` or `and` in ruby.  The precedence is different than `||` and it can lead to unexpected outcomes when checking booleans.

Reply:
Got it, thanks. Good catch!